tests/OSPSuite.FuncParser.Tests/DimensionParserSpecs.cs

```diff
@@ -3,9 +3,9 @@
 using OSPSuite.BDDHelper;
 using OSPSuite.BDDHelper.Extensions;
 
-namespace OSPSuite.FuncParser.DimensionParserTests
+namespace OSPSuite.FuncParser
```

Comment:
why did u rename the namespace? Having it like this makes separation of tests more clear and I would like to keep it like this.
![grafik](https://user-images.githubusercontent.com/25061876/61882619-27d6f780-aef9-11e9-9072-1c0add895c7d.png)